src/main/java/amazin/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,70 @@
+package amazin.controller;
+
+import amazin.model.Book;
+import amazin.model.Item;
+import amazin.model.ShoppingCart;
+import amazin.model.User;
+import amazin.repository.BookRepository;
+import amazin.repository.ShoppingCartRepository;
+import amazin.repository.UserRepository;
+import amazin.service.ShoppingCartService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+import javax.validation.Valid;
+import java.lang.reflect.Array;
+import java.security.Principal;
+import java.util.*;
+
+/**
+ * Created by lauramachado on 2019-03-18.
+ */
+@Controller
+@SessionAttributes("cart")
+public class ShoppingCartController {
+
+    @Autowired
+    ShoppingCartService shoppingCartService;
+
+    @Autowired
+    ShoppingCartRepository cartRepository;
+
+    @Autowired
+    UserRepository userRepository;
+
+    public static final String VIEW_SHOPPING_CART = "shopping-cart";
+
+    @GetMapping("/cart/delete/{bookId}")
+    public String deleteCartItem(@PathVariable("bookId") Long bookId, Model model){
```

Comment:
This should be a POST request

Reply:
I followed the same logic as the delete book in BookController

Reply:
that is a bug then, and i will fix it right now, please do the same